### DISCUSSION TITLE
Includere font custom via css

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3892] [Q&A] [answered]
Buonasera, ho notato che il browser mi da il seguente errore quando carico un sito in cui ho importato un Google Font o altro via CDN:

`Refused to load the stylesheet 'https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap' because it violates the following Content Security Policy directive: "style-src 'self' 'sha256-pru43GdcNLwb4MwzOriCI9/9cKBzE5xeoLWHlKai1As='". Note that 'style-src-elem' was not explicitly set, so 'style-src' is used as a fallback.`

In effetti i font si vedono sul mio PC poichè li ho installati ma su altri PC non vengono mostrati...
In che modo è possibile gestire queste customizzazioni?
Grazie

**Comment by evilaliv3** [comment, marked as answer]
GlobaLeaks inibisce ogni tipi di interazione con software di terze parti per ragioni di sicurezza e compliance GDPR.

Potete caricare i font come allegato della piattaforma; caricando il file a sistema sarà riferibile tramite link interno alla piattaforma su path /s/*

**Reply by maelaborazioni** [reply]
Chiarissimo. Procedo così. Grazie mille!

**Comment by maelaborazioni** [comment]
Chiedo scusa, avrei bisogno di un paio di altre info:

1. i font posso caricarli sul sito padre o devo caricarli ogni volta su tutti i siti figli?
2. come mi riferisco alla url principale nel css, qualcosa del tipo _@import url('/s/OpenSans-Regular.ttf');_ ?

**Reply by maelaborazioni** [reply]
Come non detto, trovato.
mi autorispondo nel caso servisse ad altri:

1. sì, si possono caricare sul padre ed i css dei figli li vedranno
2. con questa sintessi:

`@font-face {
    font-family: 'Open Sans';
    src: url('/s/OpenSans-Regular.ttf') format('truetype');
    font-weight: normal;
    font-style: normal;
    font-display: swap;
}`
